poglink/cogs/rates.py

```diff
@@ -59,13 +59,11 @@ async def get_current_rates(url):
     async def send_embed(self, description, rates_dict, **kwargs):
         url = rates_dict.get("url", DEFAULT_URL)
         server_name = rates_dict.get("server_name", DEFAULT_SERVER_NAME)
-        server_color = int(rates_dict.get("color", DEFAULT_COLOR), base=16)
```

Comment:
Resulted in`int() can’t convert non-string with explicit base`,

Removed the explicit base

Reply:
So the explicit base was added because there's a difference between `0xFFAABB` and `"0xFFAABB"`.

With the latter string representation (as is what's returned when parsing a yaml file), `int()` can't necessarily parse to the correct integer representation. 

Rather than remove the base here, I'd update the default value to be a string

Reply:
(See current test failure for more info ;) )

Reply:
Good now I think?